tracing-attributes/Cargo.toml

```diff
@@ -39,7 +39,7 @@ async-await = []
 
 [dependencies]
 proc-macro2 = "1"
-syn = { version = "1", features = ["full", "extra-traits", "visit-mut"] }
+syn = { version = "1", default-features = false, features = ["full", "parsing", "printing", "visit-mut", "clone-impls", "extra-traits", "proc-macro"] }
```

Comment:
nit, take it or leave it: it might be worth having a comment here summarizing what features are being disabled, for future reference? not a blocker, though.